scripts/gen_server_keys.sh

```diff
@@ -44,6 +47,8 @@ openssl genrsa -des3 -passout pass:x -out server.pass.key 2048
 openssl rsa -passin pass:x -in server.pass.key -out server.key
 chmod 600 server.key
 rm server.pass.key
+# The following command does not work on a Mac
```

Comment:
It works on mine. Both files exists, do not remember doing anything specific to fix it.